#! /bin/sh /usr/share/dpatch/dpatch-run
## 99_kill_test-xvasprintf_empty_arg.dpatch by  <mstone@annuminas.home.mathom.us>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: disable test of zero-len format string

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' coreutils-8.20~/gnulib-tests/test-xvasprintf.c coreutils-8.20/gnulib-tests/test-xvasprintf.c
--- coreutils-8.20~/gnulib-tests/test-xvasprintf.c	2012-10-23 10:17:25.000000000 -0400
+++ coreutils-8.20/gnulib-tests/test-xvasprintf.c	2012-11-13 22:27:04.391789074 -0500
@@ -94,11 +94,13 @@
 
   {
     /* Silence gcc warning about zero-length format string.  */
+/* this test can't be run when hardening flags are active
     const char *empty = "";
     result = xasprintf (empty);
     ASSERT (result != NULL);
     ASSERT (strcmp (result, "") == 0);
     free (result);
+*/
   }
 
   result = xasprintf ("%s", "foo");
